### PR TITLE
feat: add convert microservice to main docker compose

### PR DESCRIPTION
**Pull request by spiridonovfed**

### CODE REVIEW

File: convert/convert/config.py [hunk 1]
```diff
@@ -29,22 +29,22 @@
...
-    annotation_service_url: Optional[str] = os.getenv("ANNOTATION_SERVICE_URL")
-    taxonomy_service_url: Optional[str] = os.getenv("TAXONOMY_SERVICE_URL")
-    keycloak_url: Optional[str] = os.getenv("KEYCLOAK_URL")
+    assets_service_url: Optional[str] = os.getenv("ASSETS_SERVICE_HOST")
+    job_service_url: Optional[str] = os.getenv("JOBS_SERVICE_HOST")
+    annotation_service_url: Optional[str] = os.getenv(
+        "ANNOTATION_SERVICE_HOST"
+    )
+    taxonomy_service_url: Optional[str] = os.getenv("TAXONOMY_SERVICE_HOST")
+    keycloak_url: Optional[str] = os.getenv("KEYCLOAK_HOST")
```
Comment (cakeinsauce) [inline convert/convert/config.py:47]: Could we rename those according to the new naming, `*_host`

File: convert/convert/config.py [hunk 2]
```diff
@@ -29,22 +29,22 @@
...
-    minio_access_key: Optional[str] = os.getenv("MINIO_ACCESS_KEY")
-    minio_secret_key: Optional[str] = os.getenv("MINIO_SECRET_KEY")
+    minio_host: Optional[str] = os.getenv("S3_ENDPOINT_URL")
+    minio_access_key: Optional[str] = os.getenv("S3_ACCESS_KEY")
+    minio_secret_key: Optional[str] = os.getenv("S3_SECRET_KEY")
     s3_prefix: Optional[str] = os.getenv("S3_PREFIX")
-    s3_credentials_provider: Optional[str] = os.getenv(
-        "S3_CREDENTIALS_PROVIDER", "minio"
-    )
+    s3_credentials_provider: Optional[str] = os.getenv("S3_PROVIDER", "minio")
```
Comment (cakeinsauce) [inline convert/convert/config.py:36]: Could we rename these according to the new naming of env vars?

File: convert/convert/config.py (hunk 1 above)
Comment (spiridonovfed) [reply convert/convert/config.py]: Renamed

File: convert/convert/config.py (hunk 2 above)
Comment (spiridonovfed) [reply convert/convert/config.py]: Renamed

**Review by cakeinsauce — APPROVED** [review]
Great Job!

**Pull request merged**